src/components/ftbot/TradeListNav.vue

```diff
@@ -11,10 +11,7 @@
         :active="trade.open_timestamp === selectedTrade.open_timestamp"
         @click="onTradeSelect(trade)"
       >
-        <div>
-          <DateTimeTZ :date="trade.open_timestamp" />
-        </div>
-
+        <div>{{ trade.is_short ? '[S]-' : '[L]-' }}<DateTimeTZ :date="trade.open_timestamp" /></div>
```

Comment:
We may want to hide this for spot bots (this info is really irrelevant for such bots).

In backtesting scenarios - this can be checked via `botStore.activeBot.botState.trading_mode !== 'spot'` ...